layouts/partials/card.html

```diff
@@ -40,7 +40,7 @@
         </div>
       {{ else if .Params.icon }}
         <div class="hover-card-icon">
-          {{ partial "imgproc.html" (dict "src" (.Params.icon) "resize" "400x" "alt" (.Params.linkTitle) "style" "") }}
+          {{ partial "imgproc.html" (dict "src" (index .Params.images 0) "resize" "400x" "alt" (.Params.linkTitle) "style" "") }}
```

Comment:
If this is changing to be identical to line 33, how about we get rid of icon altogether? The no imgAlt case is already covered by the recent changes so maybe we don't need it anymore?

Reply:
I see that we use class `hover-card` but do we need `hover-card-icon` anymore? It appears it's used with no-js situations? Is the specific css for it necessary or can we merge the two?

Reply:
We do still need the `hover-card-icon` class, the icons get treated a tiny bit differently in css. `object-fit` changes to contain instead of `cover` which makes the components page have cards that contain the entire icon instead of being zoomed in. On smaller screens it also makes icons take up 30% of the card width instead of 50%. It's not an image so it doesn't need to be that big. It looks odd to me with icons that big, that's why they're treated differently.

Reply:
This is the difference
<img width="390" alt="Screenshot 2024-02-22 at 11 42 20" src="https://github.com/viamrobotics/docs/assets/5212232/e31b1076-88c2-4b72-b1c8-cf1f26764fa3">

<img width="386" alt="Screenshot 2024-02-22 at 11 42 35" src="https://github.com/viamrobotics/docs/assets/5212232/133acd07-d94c-43f1-8877-18fe4ec5945d">

Reply:
Okay! When I built the site before some updates here, I saw that the generated html didn't seem to include hover-card-icon anywhere but now I see it's using it again after the recent commits!